autoPyTorch/pipeline/components/setup/network_head/no_head.py

```diff
@@ -20,7 +20,7 @@ class NoHead(NetworkHeadComponent):
     """
 
     def build_head(self, input_shape: Tuple[int, ...], output_shape: Tuple[int, ...]) -> nn.Module:
-        layers = [nn.Flatten()]
+        layers = []
         in_features = np.prod(input_shape).item()
         out_features = np.prod(output_shape).item()
         layers.append(_activations[self.config["activation"]]())
```

Comment:
hey, just for correctness, could you set the 'handles_image' and 'handles_time_series' flag to false?

Reply:
Good point, we should actually investigate this in the future if it is indeed a problem to have a flatten layer or not, with all the problems there were already in there, maybe it was confusing. We should open an issue about it.

Reply:
yes, I'll make one